packs/data/equipment.db/anointed-waterskin.json

```diff
@@ -6,7 +6,7 @@
         "baseItem": null,
         "containerId": "null",
         "description": {
-            "value": "<p>This waterskin coruscates with holy energy, causing it to slowly fill itself with special blessed water unique to the item. After using any of the activations, the waterskin is empty, but slowly refills itself. It becomes full enough to use again at the next dawn.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"pf2-icon\">2</span> command, Interact</p>\n<p><strong>Requirements</strong> The anointed waterskin is full</p>\n<hr />\n<p><strong>Effect</strong> You throw the anointed waterskin up to 60 feet. The water explodes out of it, with the effects of <em>@UUID[Compendium.pf2e.spells-srd.Holy Cascade]{Holy Cascade}</em>.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> Interact</p>\n<p><strong>Requirements</strong> The anointed waterskin is full</p>\n<hr />\n<p><strong>Effect</strong> You drink the water within the waterskin, granting you the effects of <em>@UUID[Compendium.pf2e.spells-srd.Bless]{Bless}</em>. As normal with the spell, you can increase the radius with a single action with the concentrate trait.</p>\n<hr />\n<p><strong>Activate</strong> 1 minute (command, Interact)</p>\n<p><strong>Requirements</strong> The anointed waterskin is full</p>\n<hr />\n<p><strong>Effect</strong> You decant the water, creating up to 10 vials of <em>@UUID[Compendium.pf2e.equipment-srd.Holy Water]{Holy Water}</em>. You must provide the @UUID[Compendium.pf2e.equipment-srd.Vial]{Vials}.</p>"
+            "value": "<p>This waterskin coruscates with holy energy, causing it to slowly fill itself with special blessed water unique to the item. After using any of the activations, the waterskin is empty, but slowly refills itself. It becomes full enough to use again at the next dawn.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"pf2-icon\">2</span> command, Interact</p>\n<p><strong>Requirements</strong> The anointed waterskin is full</p>\n<hr />\n<p><strong>Effect</strong> You throw the anointed waterskin up to 60 feet. The water explodes out of it, with the effects of <em>@UUID[Compendium.pf2e.spells-srd.Holy Cascade]{Holy Cascade}</em>.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> Interact</p>\n<p><strong>Requirements</strong> The anointed waterskin is full</p>\n<hr />\n<p><strong>Effect</strong> You drink the water within the waterskin, granting you the effects of <em>@UUID[Compendium.pf2e.spells-srd.Bless]{Bless}</em>. As normal with the spell, you can increase the radius with a single action with the concentrate trait.</p>\n<hr />\n<p><strong>Activate</strong> 1 minute (command, Interact)</p>\n<p><strong>Requirements</strong> The anointed waterskin is full</p>\n<hr />\n<p><strong>Effect</strong> You decant the water, creating up to 10 vials of <em>@UUID[Compendium.pf2e.equipment-srd.Holy Water]{Holy Water}</em>. You must provide the vials.</p>"
```

Comment:
This needs a migration to prevent the broken link. But I'd also wager it's not a terribly common item.

Reply:
Yeah, I wasn't too worried about it. If you think a migration is necessary, I'll add one to the PR.

Reply:
I think we're fine probably. Maybe double digit people have this item on their sheet.